tensorflow_lite_support/java/src/java/org/tensorflow/lite/support/image/BoundingBoxUtil.java

```diff
@@ -197,6 +197,42 @@ private static RectF convertFromBoundaries(
     }
   }
 
+  private static RectF convertFromUpperLeft(
+      float[] values, CoordinateType coordinateType, int height, int width) {
+    if (coordinateType == CoordinateType.PIXEL) {
+      float left = values[0];
+      float top = values[1];
+      float w = values[2];
+      float h = values[3];
+
+      return new RectF(
```

Comment:
Please format it correctly based on https://google.github.io/styleguide/javaguide.html, like

```
RectF(left, top, left + w, top + h);
```

So do the rest of code, mostly binary operators.